amivapi/tests/blacklist/test_emails.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# license: AGPLv3, see LICENSE for details. In addition we strongly encourage
+#          you to buy us beer if we meet and you like the software.
+"""Tests for blacklist resource."""
+
+from amivapi.tests.utils import WebTest
+from datetime import datetime
+from freezegun import freeze_time
+
+
+class BlacklistEmailTest(WebTest):
+    """Test blacklist emails.
+
+    Everytime a user is added or removed from the blacklist, he should receive
+    an email.
+    """
+
+    def test_receive_email_on_new_entry_with_price(self):
+        """Test that a user receives an email if he is added to the blacklist"""
+        # Create a user
+        self.load_fixture({'users': [{'_id': 24 * "0", 'email': "bla@bla.bl"}]})
+        data = {
+            'user': 24 * "0",
+            'reason': "Test",
+            'price': 5,
+            'start_time': "2019-03-29T00:00:00Z",
+            'end_time': "2019-03-30T00:00:00Z",
+        }
+        self.api.post("/blacklist", data=data, token=self.get_root_token(),
+                      status_code=201)
+
+        # Look for sent out mail
+        mail = self.app.test_mails[0]
+        self.assertEqual(mail['receivers'], 'bla@bla.bl')
+        self.assertTrue("pay" in mail['text'])
```

Comment:
Using `self.assertIn("pay", mail['text'])` is preferable here, as it provides better error messages in case of failure.